pyproject.toml

```diff
@@ -1,6 +1,91 @@
+[tool.poetry]
+name = "phc-sdk-py"
+version = "0.32.0"
+description = "Python SDK for the LifeOmic platform"
+authors = ["LifeOmic <development@lifeomic.com>"]
+license = "MIT"
+readme = "README.md"
+repository = "https://github.com/lifeomic/phc-sdk-py"
+documentation = "https://lifeomic.github.io/phc-sdk-py/"
+packages = [{ include = "phc" }]
+classifiers = [
+  "Development Status :: 3 - Alpha",
+  "License :: OSI Approved :: MIT License",
+  "Intended Audience :: Developers",
+  "Intended Audience :: Healthcare Industry",
+  "Natural Language :: English",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Topic :: Utilities",
+]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+aiohttp = ">3.5.3,<4.0.0"
+backoff = ">=1.10.0,<2.0.0"
+nest-asyncio = ">=1.5.6,<2.0.0"
+pyjwt = ">=2.4.0,<3.0.0"
+pandas = ">=1.3.5,<2.0.0"
+funcy = ">=2.0,<3.0.0"
+lenses = ">=1.1.0,<2.0.0"
+toolz = ">=0.12.0,<1.0.0"
+pydantic = ">=1.10.2,<2.0.0"
+python-pmap = ">=2.0.0,<3.0.0"
+fastparquet = ">=0.8.1,<1.0.0"
+
+[tool.poetry.group.dev.dependencies]
+pre-commit = "2.21.0"
+black = "23.3.0"
+flake8 = "5.0.4"
+pdoc3 = "0.10.0"
+twine = "4.0.2"
+pytest = "7.4.0"
+pytest-watch = "4.2.0"
+click = "8.0.2"
+boto3 = "1.26.158"
+fire = "0.5.0"
+typing-extensions = "4.6.3"
+datamodel-code-generator = "0.21.1"
+poethepoet = "^0.21.1"
+
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poe.tasks]
+clean = "rm -rf build dist phc.egg-info doc/build"
+_doc_generate = """
+pdoc --html --output-dir doc/build --force --template-dir doc/template
+  --config show_inherited_members=True
+  --config list_class_variables_in_index=False
+  ./phc
+"""
+_doc_copy = "cp doc/phc.png doc/build/phc/phc.png"
+_doc_favicon = "cp doc/favicon.ico doc/build/phc/favicon.ico"
+doc = ["_doc_generate", "_doc_copy", "_doc_favicon"]
```

Comment:
Have you verified locally that `poetry run poe doc` can successfully generate the docs site?

Reply:
It is a good point to include doc generation in github action workflow. let me add it

Reply:
This is interesting: the doc build works in py3.8 and py3.9 but not py3.10+ let me dig more

Reply:
let me exclude the doc process compatibility in a separate PR to focus on using poetry only